### PR TITLE
Added A Company: BlackBox Technologies

### PR DESCRIPTION
**Pull request by ankman007**
## What kind of change does this PR introduce? 
Added BlackBox Technologies Pvt. Ltd. to the companies.json and it's details

- [ ] Bug Fix
- [ ] Documentation change
- [X] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -766,8 +766,24 @@
         "instagram": null,
         "linkedin": "https://www.linkedin.com/company/yco-pvt.-ltd./",
         "twitter": null,
-        "youtube": null,
+        "youtube": null
       }
     },   
+    {
+      "name": "BlackBox Technologies Pvt. Ltd.",
+      "establishedAt": 2018,
```
Comment (thenishantsapkota) [inline data/companies.json:774]: The `establishedAt` is a unix timestamp. Please read CONTRIBUTING.md before opening a PR.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/companies.json [hunk 2]
```diff
@@ -766,8 +766,24 @@
...
+      "location": "Kupondole, Lalitpur, Nepal",
+      "website": "https://www.bboxtech.com/",
+      "email": "info@bboxtech.com",
+      "phone": "5441320",
```
Comment (susanhsrestha) [inline data/companies.json:778]:
```suggestion
      "phone": "+977-15441320",
```

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

File: data/companies.json [hunk 3]
```diff
@@ -766,8 +766,24 @@
...
         "linkedin": "https://www.linkedin.com/company/yco-pvt.-ltd./",
         "twitter": null,
-        "youtube": null,
+        "youtube": null
       }
     },   
+    {
+      "name": "BlackBox Technologies Pvt. Ltd.",
+      "establishedAt": 2018,
+      "location": "Kupondole, Lalitpur, Nepal",
```
Comment (TheLearneer) [inline data/companies.json:775]: To be consistent with the new changes, please update as follow
```suggestion
      "location": "Kupondole, Lalitpur",
```

**Review by TheLearneer — CHANGES_REQUESTED** [review]

File: data/companies.json (hunk 1 above)
Comment (ankman007) [reply data/companies.json:774]: Oh my bad! I will fix it ASAP. Thanks.

Comment (ankman007) [reply data/companies.json]: @thenishantsapkota I've changed `establishedAt` to unix timestamp.

File: data/companies.json [hunk 4]
```diff
@@ -771,11 +771,11 @@
     },   
     {
       "name": "BlackBox Technologies Pvt. Ltd.",
-      "establishedAt": 2018,
+      "establishedAt": 1540607682000,
       "location": "Kupondole, Lalitpur, Nepal",
       "website": "https://www.bboxtech.com/",
       "email": "info@bboxtech.com",
-      "phone": "5441320",
+      "phone": "01-5541320",
```
Comment (susanhsrestha) [inline data/companies.json:778]: The format of the phone number is not correct.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please format the data correctly in alphabetical order before pushing

**Review by thenishantsapkota — COMMENTED** [review]
Also make sure to incorporate the latest changes into your PR.

**Comment by ankman007** [comment]
> Please format the data correctly in alphabetical order before pushing

Done

File: data/companies.json [hunk 5]
```diff
@@ -127,6 +127,22 @@
...
         "youtube": null
       }
+    },  
+    {
+      "name": "BlackBox Technologies Pvt. Ltd.",
+      "establishedAt": 1540607682000,
+      "location": "Kupondole, Lalitpur",
+      "website": "https://www.bboxtech.com/",
+      "email": "info@bboxtech.com",
+      "phone": "01-5541320",
```
Comment (thenishantsapkota) [inline data/companies.json:137]: The phone number format isn't correct. 
```suggestion
      "phone": "+977-15541320",
```

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by susanhsrestha — COMMENTED** [review]
The format for the phone number is incorrect.

File: data/companies.json [hunk 6]
```diff
@@ -127,6 +127,22 @@
...
         "youtube": null
       }
+    },  
+    {
+      "name": "BlackBox Technologies Pvt. Ltd.",
+      "establishedAt": 1540607682000,
+      "location": "Kupondole, Lalitpur",
+      "website": "https://www.bboxtech.com/",
+      "email": "info@bboxtech.com",
+      "phone": "+941-6290032",
```
Comment (thenishantsapkota) [inline data/companies.json:137]: Again the format is incorrect

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by thenishantsapkota — APPROVED** [review]
LGTM!

**Review by TheLearneer — APPROVED** [review]
LGTM

**Pull request merged**